### ISSUE TITLE
Bug? ECHO doesn't escape strings in a vector

### ISSUE THREAD

**Issue by MichaelAtOz** [openscad/openscad#2078]
```
t=["first line\nsecond line"];
echo(t);
echo(t[0]);
```
Produces
```
Compiling design (CSG Tree generation)...
ECHO: ["first line\nsecond line"]
ECHO: "first line
second line"
Compiling design (CSG Products generation)...
```
Issue #838 may relate.

**Comment by thehans** [comment]
@MichaelAtOz I commited a fix for this, can you try a nightly build?

**Comment by MichaelAtOz** [comment]
@t-paul is a ludite like me likely to have the the ability to do a nightly build for Windows?

**Comment by t-paul** [comment]
Fixed by #2126.

**Comment by t-paul** [comment]
Building Windows snapshots is mostly automated but manually started, but uses a Linux based environment ([mxe.cc](http://mxe.cc/)). I wonder if there's a way to set the build up on Travis for really automatic build and upload to the snapshot folder.
